pkg/cloud/azure/services/config/controlplane.go

```diff
@@ -53,7 +54,6 @@ echo -n '{{.SaKey}}' > /etc/kubernetes/pki/sa.key
 chmod 600 /etc/kubernetes/pki/sa.key
 
 PRIVATE_IP=$(curl -H Metadata:true "http://169.254.169.254/metadata/instance/network/interface/0/ipv4/ipAddress/0/privateIpAddress?api-version=2018-10-01&format=text")
-HOSTNAME="$(curl -H Metadata:true "http://169.254.169.254/metadata/instance/compute/name?api-version=2018-10-01&format=text")"
```

Comment:
We still need the hostname as part of the SAN.

Reply:
Its already taken care of by kubeadm, which uses current computer name.